spec/support/contexts/response_tree.rb

```diff
@@ -54,4 +54,55 @@ def web_answer_group_hash(q_id, children, relevant: "true", destroy: nil, id: ""
     hash[:_destroy] = destroy unless destroy.nil?
     hash
   end
+
+  def path_selector(indices, suffix = "value")
+    path = ["children"] + indices.zip(["children"] * (indices.length - 1)).flatten.compact
+    "response_root_#{path.join('_')}_#{suffix}"
+  end
+
+  def fill_in_question(path, opts)
```

Comment:
How does this method compare with `fill_answer` in `hierarchical_response_form_conditional_logic.rb`? Can we combine them perhaps?

Reply:
I think we can combine them once we refactor `hierarchical_response_display_logic_spec.rb` to handle form items hierarchically (instead of by type as it is now).  I believe you made a separate ticket to handle this work and I can do that next (new PR).

Reply:
Ok can you put a big fat TODO please explaining what can be combined and how? We may not get to it for awhile.

Reply:
Should I commit the comment to `develop`?  I see this was already merged